modules/ossm-running-v2-v3-cluster-wide-deployment-model.adoc

```diff
@@ -0,0 +1,190 @@
+[source, yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  mode: ClusterWide
+----
+
+To prevent conflicts with {SMProduct} 3.0 when using the {SMProduct} 2.6 cluster-wide deployment model, you need to configure the `ServiceMeshControlPlane` resource to restrict namespaces to only those belonging to (SMProduct) 2.6.
+
+//NOTES: 10/23/2024
+
+//DO NOT USE VERSION NUMBER ATTRIBUTE. It is set for 3.0, will likely change, and it is unclear if later versions would impact this procedure for moving from 2.6.
+//Only moving from 2.6 has been tested and verified by QE.
+//Cluster-wide requires changes to OSSM 2.6 ServiceMeshControlPlane before installing OSSM 3.0
+
+.Prerequisites
+
+* You are running {ocp-product-title} 4.14 or later.
+* You are running {SMProduct} 2.6.
+
+If you are not running {SMProduct} 2.6, you must upgrade to 2.6 before following this procedure. To upgrade from you current {SMProduct} version to 2.6, see: link:https://docs.openshift.com/container-platform/4.17/service_mesh/v2x/upgrading-ossm.html[Upgrading Service Mesh 2.x]
+
+.Procedure
+
+//To better understand the steps described here, it's recommended to read the OSSM 2 [deployment models](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-deployment-models) and upstream Istio [Multiple Control Planes](https://istio.io/latest/docs/setup/install/multiple-controlplanes/) documentation. This document assumes OSSM 2.6 is already running and the goal is to install OSSM 3.0 on the same cluster without interfering with the OSSM 2.6 installation.
+
+//Follow the steps described below for your OSSM 2 deployment model (Multi-tenant or Cluster-Wide).
+
+. Configure `discoverySelectors` in your {SMProduct} 2.6 `ServiceMeshControlPlane` custom resource (CR):
++
+.Example `ServiceMeshControlPlane` CR
+[source, yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+  spec:
+    policy:
+       type: Istiod
+    addons:
+      grafana:
+        enabled: false
+      kiali:
+        enabled: true
+      prometheus:
+         enabled: true
+    telemetry:
+      type: Istiod
+    version: v2.6
+    mode: ClusterWide
+    meshConfig:
+      discoverySelectors:
+        - matchExpressions:
+          - key: maistra.io/member-of
+            operator: Exists
+----
+
+. Set the `ENABLE_ENHANCED_RESOURCE_SCOPING ` environment variable on the pilot container to `true` in your {SMProduct} 2.6 `ServiceMeshControlPlane` CR:
++
+.Example `ENABLE_ENHANCED_RESOURCE_SCOPING` in `ServiceMeshControlPlane` CR
+[source, yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+  spec:
+    runtime:
+      components:
+        pilot:
+          container:
+            env:
+              ENABLE_ENHANCED_RESOURCE_SCOPING: 'true'
+----
```

Comment:
IMO this step and the step above should be combined into a single step and not two separate steps since it's the same YAML. If it's two separate steps then you'd need to edit/apply it twice.

Full YAML:
```
apiVersion: maistra.io/v2
kind: ServiceMeshControlPlane
metadata:
  name: basic
  namespace: istio-system
spec:
  version: v2.6
  mode: ClusterWide
  meshConfig:
    discoverySelectors:
      - matchExpressions:
        - key: maistra.io/member-of
          operator: Exists
  runtime:
    components:
      pilot:
        container:
          env:
            ENABLE_ENHANCED_RESOURCE_SCOPING: 'true'
```